examples/llava/mtmd-cli.cpp

```diff
@@ -206,6 +215,7 @@ static int eval_message(mtmd_cli_context & ctx, common_chat_msg & msg, std::vect
     LOG_DBG("formatted_chat.prompt: %s\n", formatted_chat.prompt.c_str());
 
     for (auto & fname : images_fname) {
+        if (g_is_interrupted) return 0;
```

Comment:
I think this is redundant because this loop is very fast.

The call that takes the most time is `mtmd_helper_eval`